homeassistant/components/mqtt/__init__.py

```diff
@@ -399,19 +395,6 @@ async def async_forward_entry_setup_and_setup_discovery(
             )
         # Setup reload service after all platforms have loaded
         await async_setup_reload_service()
-        # When the entry is reloaded, also reload manual set up items to enable MQTT
-        if mqtt_data.reload_entry:
-            mqtt_data.reload_entry = False
-            reload_manual_setup = True
-
-        # When the entry was disabled before, reload manual set up items to enable
-        # MQTT again
-        if mqtt_data.reload_needed:
-            mqtt_data.reload_needed = False
-            reload_manual_setup = True
-
-        if reload_manual_setup:
-            await async_reload_manual_mqtt_items(hass)
```

Comment:
Can `homeassistant/components/mqtt/__init__.py:async def async_reload_manual_mqtt_items(hass: HomeAssistant) -> None:` be removed as well?

Reply:
Apparently not used methods do not always show up in pytest coverage reports. Nice cleanup I would say.